spec/p2p/reactors/mempool-v1.md

```diff
@@ -0,0 +1,81 @@
+<!-- The allocation of max_num_inbound_peers and max_num_outbound_peers across various connection types requires clarification. For instance, whether max_num_inbound_peers includes both unconditional peers and persistent peers or not. -->
+
+## Traffic Rate Analysis
+In the analysis provided below, we consider the knowledge of the following network parameters
+- `d`: Node degree (total incoming and outgoing connections)
+<!-- - transaction rate: `transaction_rate` total number of transactions per second submitted to the network -->
+- `transaction_rate` which specifies that total size of transactions in bytes per second submitted to the network.
+- `C`: total number of connections in the network
+
+Transactions are assumed to comply with the transaction size, are valid and are accepted by the mempool. We also assume all the peers are up and running.
+
+### Traffic Rate Analysis for a Node
+We distinguish between the incoming and outgoing traffic rate, and denote them by  `incoming_traffic_rate` and  `outgoing_traffic_rate`, respectively.
+- **Worst case scenario**: a transaction is exchanged by the two ends of
+connection simultaneously, contributing to both incoming and outgoing traffic.
+In a network, with transaction rate `transaction_rate` and a node with `d` degree, the traffic rates are calculated as follows:
+`incoming_traffic_rate = d * transaction_rate`
+`outgoing_traffic_rate = d * transaction_rate`
+
+These max rates are further constrained by the `SendRate` and `RecRate`.
+`incoming_traffic_rate = d * min(transaction_rate, SendRate)`
+`outgoing_traffic_rate = d * min(transaction_rate, RecRate)`
+
+- **Best case scenario**: a transaction is exchanged only once, contributing to either incoming or outgoing traffic. This is because both ends of the connection keep track of the transactions they have seen on a connection (whether via sending or receiving). If one peer sends a transaction before the other, they both mark it as sent/received, ensuring they do not redundantly transmit it to each other.
+In a network, with transaction rate `transaction_rate` and a node with  degree of `d`, the node's traffic rate in best case would be:
+`traffic_rate (=incoming_traffic_rate + outgoing_traffic_rate) = d * transaction_rate`
+
+We can draw the following conclusions (to be extended and verified):
+- With a known given transaction rate `transaction_rate`, a node's (in + out) traffic rate should range from `d * transaction_rate` to `2 * d * transaction_rate`.
+- To handle a particular `transaction_rate` (network throughput), the node's `SendRate` and `RecRate` should be at least `transaction_rate` to handle the worst case scenario (this is only to undertake the load incurred by the mempool reactor).
```

Comment:
[nit] here `RecRate` is used but it appears defined as `RecvRate` above. Proposal for it to be consistent.

Reply:
Fixed https://github.com/celestiaorg/celestia-core/pull/1108/commits/ff9dc269b9dd74a28920470774e011fe6c7db71d.